.github/workflows/nightly-rosetta-t5x-build-test.yaml

```diff
@@ -148,6 +108,42 @@ jobs:
       EXPERIMENT_SUBDIR: ROSETTA_T5X
     secrets: inherit
 
+  publish-test:
+    needs: [metadata, build, test-unit, test-t5x]
+    uses: ./.github/workflows/_publish_badge.yaml
+    if: ( always() )
+    secrets: inherit
+    with:
+      ENDPOINT_FILENAME: 'rosetta-t5x-test-status.json'
```

Comment:
Maybe to just be explicit, we can name this `rosetta-t5x-overall-test-status.json`